users.json

```diff
@@ -1,47 +1,38 @@
 [
   {
+    "_id":"63e8c794ac6688d05d7294d3",

```

Comment:
The value of _id is given by mongo, would it be better to delete the hardcode _id? And the other id too, because we're not using it.

Reply:
This value is used to seed the database so we need to be specific about it to be able to set relationships between the users (following) 

If we let Mongo decide the value on its own, every time one of us seeds the database the IDs will be different and the followings will be invalid